templates/workloads/partials/cf/envvars.yml

```diff
@@ -14,4 +14,10 @@ Environment:
   Value: {{$value | printf "%q"}}{{end}}{{end}}{{if .NestedStack}}{{$stackName := .NestedStack.StackName}}{{range $var := .NestedStack.VariableOutputs}}
 - Name: {{toSnakeCase $var}}
   Value:
-    Fn::GetAtt: [{{$stackName}}, Outputs.{{$var}}]{{end}}{{end}}
\ No newline at end of file
+    Fn::GetAtt: [{{$stackName}}, Outputs.{{$var}}]{{end}}{{end}}
+{{- if .Storage}}
+{{- range $n, $mp := .Storage.MountPoints}}
+- Name: MOUNT_PATH_{{$n}}
```

Comment:
My memory is terrible T_T sorry, is this information useful for folks in their application? or is it strictly for us to use in `svc show`?

If it's the latter can we  get that information from describing the task defs instead?

Reply:
This info is important, as services may need to know this location in order to perform IO on the mounted EFS volume